tests/main/interfaces-fuse_support/task.yaml

```diff
@@ -1,5 +1,7 @@
 summary: Ensure that the fuse-support interface works.
 
+systems: [-ubuntu-14.04-64]
+
```

Comment:
What would be the best forward? Only conditionally include it in interfaces/builtin/all.go?